Makefile

```diff
@@ -41,13 +41,18 @@ generate_builtins_test::
 
 
 tidy::
-	cd pf && go mod tidy
-	cd pf/tests && go mod tidy
-	cd pf/tests/internal/randomshim && go mod tidy
-	cd pf/tests/internal/tlsshim && go mod tidy
-	cd pf/tests/testdatagen/genrandom && go mod tidy
-	cd pkg/tests && go mod tidy
-	cd x/muxer && go mod tidy
-	cd x/muxer/tests && go mod tidy
-	cd testing && go mod tidy
-	go mod tidy
+	find . -name go.mod -execdir go mod tidy \;
```

Comment:
Nice, good shel-fu . I should stop writing Go programs for something of this nature.